smoke-tests-otel-starter/spring-boot-common/src/main/java/io/opentelemetry/spring/smoketest/AbstractJvmMongodbSpringStarterSmokeTest.java

```diff
@@ -29,7 +29,7 @@ public class AbstractJvmMongodbSpringStarterSmokeTest
 
   @BeforeAll
   static void setUpContainer() {
-    container = new MongoDBContainer("mongo:4.0");
+    container = new MongoDBContainer("mongo:4.2");
```

Comment:
Does updating the mongodb driver version here is required for this temporary work-around or is it just because you've tried first to update it to a newer version to see if that helped to fix the issue ? This might be something to keep and not revert when we remove this temporary work-around in the future.

Reply:
it is required, latest dep tests update the java driver and it refuses to speak with versions earlier than 4.2